suites/portal/coremetadatapageTest.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+Feature('Core Metadata');
+
+Scenario('test core metadata', function* (I) {
```

Comment:
This is a good test! I like how you abstracted away navigation to the page. But we need a few more Scenarios probably to cover all of the cases. Are there special cases, ie the file was not found, or I don't have permission to access it, or there is no data in the project? Also I wonder if they wanted you to test the filters on the side, not sure, probably should ask about that.